include/deal.II/lac/sparsity_tools.h

```diff
@@ -79,7 +90,9 @@ namespace SparsityTools
    */
   void partition (const SparsityPattern     &sparsity_pattern,
                   const unsigned int         n_partitions,
-                  std::vector<unsigned int> &partition_indices);
+                  std::vector<unsigned int> &partition_indices,
+                  const Partitioner partitioner = Partitioner::metis
```

Comment:
same here, please adjust the documentation of the function to reflect new functionality.